lib/models/addons-factory.js

```diff
@@ -44,6 +48,11 @@ var AddonsFactory = CoreObject.extend({
     graph.topsort(function (vertex) {
       var addonInfo = vertex.value;
       if (addonInfo) {
+        var token = heimdall.start({
```

Comment:
Maybe rename the tokens in this file to `initializeAddonsToken` and `initializeAddonToken`?  Just something to prevent the shadowing here which is (a little bit) confusing.